plugins/modules/sefcontext.py

```diff
@@ -39,21 +41,28 @@
     default: a
   setype:
     description:
-    - SELinux type for the specified target.
+    - SELinux type for the specified I(target).
+    type: str
+  equal:
```

Comment:
It is the switch name used by `semanage`. But I agree it is maybe not the best name and I would absolutely like to change it to something better.

However `target_substitute` sounds a bit clumsy to me. How about just `substitute`? Or `substituted`?

Reply:
I believe they are supposed to be nouns (rather than verbs), if that's what you're asking. Other than that, I will trust any choice you make more than I would mine - I am not an user of `SELinux` tools.

Reply:
Substitute would be a noun. I'll leave this open for now and hopefully someone else will pitch in also.  
Otherwise I'll go with `substitute` - it is at least better than `equal`. Thanks for the idea.

Reply:
I went with `substitute` - thanks for the suggestion I think this is much better than `equal`.